StorageUtility/Windows/Device.cpp

```diff
@@ -52,6 +52,11 @@ DeviceHandle cDevice::Handle()
     return deviceHandle;
 }
 
+void cDevice::DevicePath(tchar*& DevicePath)
+{
+    DevicePath = m_DevDetailData->DevicePath;
```

Comment:
the DevicePath will be retrieved directly from the m_DevDetailData (PSP_INTERFACE_DEVICE_DETAIL_DATA)